io.github.cmus.cmus.yml

```diff
@@ -0,0 +1,134 @@
+  - --socket=pulseaudio
+  - --filesystem=xdg-music
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --own-name=org.mpris.MediaPlayer2.cmus
+  # Audio CD access
+  - --device=all
+
+modules:
+  - name: faad2
+    cleanup:
+      - /share/man
+      - /include
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/knik0/faad2/archive/2_10_0.tar.gz
+        sha512: 92c8b5e79f85b7a7caac9e7954959c26d74ef148a658dcb37e2c1b303fc2fcc5a0d12f21e7bd476870498109bdd03916a56c13f578186a69d6eceb103222e771
+
+  - name: libmad
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: ftp://ftp.mars.org/pub/mpeg/libmad-0.15.1b.tar.gz
+        sha512: 2cad30347fb310dc605c46bacd9da117f447a5cabedd8fefdb24ab5de641429e5ec5ce8af7aefa6a75a3f545d3adfa255e3fa0a2d50971f76bc0c4fc0400cc45
+      - type: patch
+        path: libmad-optimize.diff
+      - type: file
+        path: config.guess
+
+  - name: libmp4v2
+    cleanup:
+      - /share/man
+      - /include
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/TechSmith/mp4v2/archive/Release-ThirdParty-MP4v2-4.1.3.tar.gz
+        sha512: 6cb63997ae7a162f91f11872f5a42e10305cf57fe52989650c8817daab94f079518668f408719d2df4e64047987679e3d82a5dd202ea69cc9fe4b7b5c84fbe11
+      - type: patch
+        path: libmp4v2-gcc7.patch
+      - type: patch
+        path: libmp4v2-gcc10.patch
+      - type: file
+        path: config.guess
+        dest-filename: autoaux/config.guess
+
+  - name: libmodplug
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://download.sourceforge.net/project/modplug-xmms/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz
+        sha512: 880e10154fd367ee24ace53ca7e32a5c1fee7dfff1f934e6499944c12779427da63e2f55f8d6ce25db0a42a5b4424053bf64305b31dbfc4ef6a8909924d655fd
+
+  # Instruction taken from org.freac.freac.json
+  - name: musepack
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - /bin
+    config-opts:
+      - LDFLAGS=-Wl,--allow-multiple-definition
+    sources:
+      - type: archive
+        url: https://files.musepack.net/source/musepack_src_r475.tar.gz
+        sha256: a4b1742f997f83e1056142d556a8c20845ba764b70365ff9ccf2e3f81c427b2b
+      - type: shell
+        commands: 
+          - touch include/config.h.in
+          - head -n 34 configure.in >> configure.ac
+          - echo "AM_CONDITIONAL([HAVE_VISIBILITY], [false])" >> configure.ac
+          - echo "AC_CHECK_LIB([m],[cos])" >> configure.ac
+          - tail -n 18 configure.in >> configure.ac
+          - rm configure.in
+          - autoreconf -i
+
+  - name: libcddb
+    cleanup:
+      - /include
+      - /bin
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/libcddb/libcddb-1.3.2.tar.bz2
+        sha512: 0e07e7d37f2013164d75b530a5528f54847271fd20f3b7bedb192a05d1f61dcf006d10dc2927efe155a01eddcc57b940bc31d8ac88d5dfc4f1a09771caa84e0a
+      - type: file
```

Comment:
You can just copy over these files like here: https://github.com/flathub/org.kde.okular/blob/909e7691e274b8d1f991c4b2e045d23bc33a416b/org.kde.okular.json#L191

Reply:
Fixed by 271adf3.